otwcode/otwarchive
# AO3-5493 Reindex tags synchronously for wrangling changes

## Issue

https://otwarchive.atlassian.net/browse/AO3-5493

## Purpose

Makes it so enqueue_to_index actually just reindexes tags synchronously, allowing the unwrangled tag bins to update as close to immediately as possible once a tag is wrangled. We chose to override the method rather than create a new one for simplicity's sake.

## Testing Instructions

Go to an unwrangled bin, move a tag to a fandom, see if it has disappeared either as soon as the page reloads (somewhat unlikely) or at least after waiting ten seconds and refreshing.
